homeassistant/components/icloud/config_flow.py

```diff
@@ -83,6 +89,7 @@ def __init__(self):
 
         self._username = user_input[CONF_USERNAME]
         self._password = user_input[CONF_PASSWORD]
+        self._with_family = user_input.get(CONF_WITH_FAMILY, DEFAULT_WITH_FAMILY)
```

Comment:
I thought this change would work but launching tests tells me not.

In case of import it should use the default if not set and apparently tests (maybe real case too) are not using the init schema.

See `CONF_MAX_INTERVAL` and `CONF_GPS_ACCURACY_THRESHOLD` which also have default in init but use `get()`

Reply:
So I reverted sorry